### DISCUSSION TITLE
Language files

### DISCUSSION THREAD

**Discussion by Elektrickser** [KlipperScreen/KlipperScreen#41] [Ideas]
Hi,

would we collect language files here. I've translatet in German and would share this?

Regrads Olli

**Comment by Elektrickser** [comment]
I can't open the new KlipperScreen.pot in POedit. 
If I open in Notepad++ i see many differents. The new file is in html-Format. 

Is this correct?

Regards Olli

**Reply by Elektrickser** [reply]
ok, can closed. wrong Download. Now, it works

**Comment by jordanruthe** [comment]
Feel free to submit a github pull request, or attach them here.
